pipelines/imputation/scientificValidation/ReshapeReferencePanel.wdl

```diff
@@ -4,21 +4,508 @@ version 1.0
 workflow ReshapeReferencePanel {
     input {
         File ref_panel_vcf
+        File ref_panel_vcf_index
+        String output_base_name
```

Comment:
the underscore between base and name is a lil weird

Reply:
yea will rename, its basename everywhere else in the file